docs/source/general/contributing.rst

```diff
@@ -10,17 +10,24 @@ Follow these guidelines to speed up the process.
   :backlinks: none
   :depth: 3
 
+Note for contributors
+---------------------
+
+Before continuing with your contributing endeavours, please read the
+OpenWISP contributing guidelines to ensure smooth working of the
+freeradius repository.
+
+The guidelines can be found at the following link:
+<http://openwisp.io/docs/developer/contributing.html>
+
 Reach out before you start
```

Comment:
This is repeated, can you please remove this section.

Reply:
@Aviral14 this needs attention too, "Reach out before you start" step is here as well: http://openwisp.io/docs/developer/contributing.html as "Introduce yourself"